actor/echo/Makefile

```diff
@@ -0,0 +1,61 @@
+# examples/actor/echo
+
+KEYDIR ?= .keys
+CARGO ?= cargo
+REVISION := 0
+RUST_DEPS := Cargo.toml Makefile $(wildcard src/*.rs) .cargo/config.toml
+TARGET := target/wasm32-unknown-unknown
+VERSION := $(shell cargo metadata --no-deps --format-version 1 | jq -r '.packages[] .version' | head -1)
+
+# project is a short slug name that will be used in the final file name
+PROJECT := echo
+# list of all contract claims for actor signing (space-separated)
+CLAIMS  := wasmcloud:httpserver
+DIST_WASM := build/$(PROJECT)_s.wasm
+
+all: release $(DIST_WASM) $(DIST_EXS)
+
+release:
+		cargo build --release
+
+wasm: build/$(PROJECT)_s.wasm
+
+$(DIST_WASM) : $(TARGET)/release/$(PROJECT)_s.wasm
+	@mkdir -p $(dir $@)
+	cp -p $< $@
+
+
+.ONESHELL:
+build/stub.exs: $(DIST_WASM)
```

Comment:
please please tell me that we're not requiring the use of elixir script execution as part of the core actor development experience. I can't tell what this is for from the surrounding context, so if we don't need this it should be removed.

Reply:
This was a band-aid I was using while waiting for manifests to become available for OTP.